README.md

```diff
@@ -130,7 +130,7 @@ For configuring Octant, setting up a development environment, or running tests,
 
 Plugins are a core part of Octant in the Kubernetes ecosystem. A plugin can read objects and allows users to add components to Octant's views.
 
-An example plugin can be found in [`cmd/octant-sample-plugin`](cmd/octant-sample-plugin) and installed to the default plugin path with `make install-test-plugin`.
+An example plugin can be found in [`cmd/octant-sample-plugin`](cmd/octant-sample-plugin) and installed to the default plugin path with `go run build.go install-test-plugin`.
 
 Documentation for plugin components can be found in [`docs/plugins`](docs/plugins).
```

Comment:
Hello,
In this line the `docs/plugins` link in invalid. It should be directed to `site/docs/plugins`.
Do you want to fix it in this pull request, or should i open another a new one for this purpose ?

Reply:
Thank you! I will fix that here too.

Reply:
@alexbrand is going to fix all of these in #414